inspirehep/modules/authors/rest/citations.py

```diff
@@ -50,13 +50,11 @@ def serialize(self, pid, record, links_factory=None):
             "match": {
                 "authors.recid": author_pid
             }
-        }).params(
-            _source=[
-                "authors.recid",
-                "control_number",
-                "self",
-            ]
-        )
+        }).params(_source=[
+            "authors.recid",
+            "control_number",
+            "self",
+        ])
```

Comment:
Feels strange that one indentation level does not match one logical level to me, for me this is clearer:

```python
).params(
    _source=[
        "authors.recid",
        "control_number",
        "self",
    ]
)
```

Even if it uses more vertical space.